templates/deployments.yaml

```diff
@@ -831,6 +871,8 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmaps_st2-conf.yaml") $ | sha256sum }}
         checksum/packs: {{ include (print $.Template.BasePath "/configmaps_packs.yaml") $ | sha256sum }}
+        {{- if $.Values.secrets.st2.datastore_crypto_key }}
```

Comment:
missing a closing `end` tag for `if`?

Reply:
In fact, do we really need `if` -> `end` for the datastore crypto key checksum in annotations?
I mean even if `secrets_datastore_crypto_key.yaml` is empty if no key is provided, when it's populated the checksum is changed.

Reply:
Yep, not quite awake yet...

Indeed, removing the conditional altogether would be cleaner, implementing.